scripts/algebra-gas/README.md

```diff
@@ -0,0 +1,48 @@
+Scripts that help generate/evaluate gas parameters for generic algebra move module.
+
+## Quickstart guide
+Ensure you are on a machine with the [required spec](https://aptos.dev/nodes/validator-node/operator/node-requirements/).
+
+Ensure you have python3 and the following dependencies.
+```
+pip3 install numpy matplotlib
+```
+
+Ensure you `cd` to the repo root.
+
+Run the necessary benches.
+```
+cargo bench -p aptos-crypto -- hash/SHA2-256
+cargo bench -p aptos-crypto -- ark_bls12_381
+```
+
+If you don't have a `gas_per_ns`, estimate one using `hash/SHA2-256` bench results.
+```
+scripts/algebra-gas/load_bench_datapoints.py --bench_path target/criterion/hash/SHA2-256
+scripts/algebra-gas/fit_linear_model.py --dataset_path hash_SHA2_256.0-1025.json --plot
+```
+You will see some output like:
```

Comment:
You mean "This will output a curve $f(x) = k*x + b$ that predicts the time to evaluate SHA2-256 on an input of size $x$."

Reply:
Fixed.